lib/python/picongpu/picmi/simulation.py

```diff
@@ -99,17 +104,28 @@ def __init__(
         self,
         picongpu_template_dir: typing.Optional[typing.Union[str, pathlib.Path]] = None,
         picongpu_typical_ppc: typing.Optional[int] = None,
+        picongpu_custom_user_input: typing.Optional[customuserinput.CustomUserInput | rendering.RenderedObject] = None,
         **kw,
     ):
-        # delegate actual work to parent
+        # delegate additional work to parent
         super().__init__(**kw)
 
-        # perform some additional checks on inputs
+        # additional checks on inputs, @todo move to PICMI
```

Comment:
Aren't we in PICMI here?

Reply:
should be move to picmistandard the reference implementation of the PICMI standard